src/sagemaker/serve/builder/model_builder.py

```diff
@@ -881,8 +881,8 @@ def _build_for_model_server(self):  # pylint: disable=R0911, R1710
         if self.model_metadata:
             mlflow_path = self.model_metadata.get(MLFLOW_MODEL_PATH)
 
-        if not self.model and not mlflow_path:
-            raise ValueError("Missing required parameter `model` or 'ml_flow' path")
+        if not self.model and not mlflow_path and not self.inference_spec:
```

Comment:
Have we done e2e testing for this?

Reply:
yes, there are screenshots that Bryannah posted in the slack channel